tests/pytest/oauth/test_views.py

```diff
@@ -8,107 +8,20 @@
-    mocked_oauth_create_client.side_effect = Exception("Side effect")
-    return mocked_oauth_create_client
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification")
-def test_oauth_client_or_error_redirect_no_oauth_client(
-    app_request,
-    model_EnrollmentFlow_with_scope_and_claim,
-    mocked_oauth_create_client,
-    mocked_view_analytics_module,
-    mocked_sentry_sdk_module,
-):
-    mocked_oauth_create_client.return_value = None
-
-    result = _oauth_client_or_error_redirect(app_request, model_EnrollmentFlow_with_scope_and_claim)
-
-    assert result.status_code == 302
-    assert result.url == reverse(routes.OAUTH_SYSTEM_ERROR)
-    mocked_view_analytics_module.error.assert_called_once()
-    mocked_sentry_sdk_module.capture_exception.assert_called_once()
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification", "mocked_oauth_client_or_error_redirect__error")
-def test_oauth_client_or_error_redirect_oauth_client_exception(
-    app_request, model_EnrollmentFlow_with_scope_and_claim, mocked_view_analytics_module, mocked_sentry_sdk_module
-):
-    result = _oauth_client_or_error_redirect(app_request, model_EnrollmentFlow_with_scope_and_claim)
-
-    assert result.status_code == 302
-    assert result.url == reverse(routes.OAUTH_SYSTEM_ERROR)
-    mocked_view_analytics_module.error.assert_called_once()
-    mocked_sentry_sdk_module.capture_exception.assert_called_once()
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification", "mocked_oauth_create_client")
-def test_oauth_client_or_error_oauth_client(
-    app_request, model_EnrollmentFlow_with_scope_and_claim, mocked_view_analytics_module, mocked_sentry_sdk_module
-):
-    result = _oauth_client_or_error_redirect(app_request, model_EnrollmentFlow_with_scope_and_claim)
-
-    assert hasattr(result, "authorize_redirect")
-    mocked_view_analytics_module.error.assert_not_called()
-    mocked_sentry_sdk_module.capture_exception.assert_not_called()
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification", "mocked_oauth_client_or_error_redirect__error")
-def test_login_oauth_client_init_error(client, mocked_hook_analytics_module, mocked_hook_sentry_sdk_module):
-    path = reverse(routes.OAUTH_LOGIN)
-    response = client.get(path)
-
-    assert response.status_code == 302
-    assert response.url == reverse(routes.OAUTH_SYSTEM_ERROR)
-    mocked_hook_analytics_module.error.assert_called_once()
-    mocked_hook_sentry_sdk_module.capture_exception.assert_called_once()
 
 
 @pytest.mark.django_db
 @pytest.mark.parametrize(
     "route",
-    [routes.OAUTH_LOGIN, routes.OAUTH_AUTHORIZE, routes.OAUTH_CANCEL, routes.OAUTH_LOGOUT, routes.OAUTH_POST_LOGOUT],
+    [
+        routes.OAUTH_LOGIN,
+        OAuthRoutes.route_authorize,
+        OAuthRoutes.route_cancel,
+        routes.OAUTH_LOGOUT,
```

Comment:
Same here, can we just use `OAuthRoutes.*` everywhere?

Reply:
Yep, thanks for catching this, updated it in 72e2f3f038fa5a73881a4b510cc4d5be1fefd444